sdk/containerregistry/azure-containerregistry/azure/containerregistry/_container_repository_client.py

```diff
@@ -44,6 +44,15 @@ def __init__(self, endpoint, repository, credential, **kwargs):
         :type credential: :class:`~azure.core.credentials.TokenCredential`
         :returns: None
         :raises: None
+
+        .. admonition:: Example:
+
+            .. literalinclude:: ../samples/sample_create_client.py
+                :start-after: [START create_registry_client]
+                :end-before: [END create_registry_client]
```

Comment:
I think it might be this?
```suggestion
                :start-after: [START create_repository_client]
                :end-before: [END create_repository_client]
```